### PR TITLE
Indonesian - Installation Guide Updates

### PR DESCRIPTION
**Pull request by buddyy93**
The previous one is outdated, so i propose this update to help other developers understand the latest nuxt

### CODE REVIEW

**Comment by manniL** [comment]
@jefrydco could you review it please? :relaxed:

File: id/guide/installation.md [hunk 1]
```diff
@@ -16,59 +16,84 @@ description: Memulai Nuxt.js itu ternyata sangat mudah. Satu proyek sederhana ha
...
-## Mulai dari awal
+## Memulai dari awal
 
-Membuat aplikasi Nuxt.js dari awal juga sangat mudah, hanya memerlukan *1 file dan 1 direktori*. Mari kita buat satu direktori kosong untuk mulai membuat aplikasinya:
+Membuat suatu aplikasi dengan Nuxt.js dari awal itu mudah, anda hanya membutuhkan _1 file dan 1 direktori_. Mari kita membuat direktor baru untuk memulai pengerjaan projek:
 
 ```bash
-$ mkdir <nama-proyek>
-$ cd <nama-proyek>
+$ mkdir <project-name>
```
Comment (jefrydco) [inline id/guide/installation.md]: I think its better if it also translated into `<nama-projek>`

File: id/guide/installation.md [hunk 2]
```diff
@@ -16,59 +16,84 @@ description: Memulai Nuxt.js itu ternyata sangat mudah. Satu proyek sederhana ha
...
-<b>Info:</b> tukar <code>&lt;nama-proyek&gt;</nom-du-projet></code> dengan nama proyek yang diinginkan.
+<b>Info:</b> ganti <code>&lt;project-name&gt;</nom-du-projet></code> dengan nama project anda.
 
 </div>
 
-### package.json
+### The package.json
 
-Proyek memerlukan file `package.json` untuk memperinci bagaimana untuk memulai `nuxt`:
+Projek anda membutuhkan file `package.json` untuk menjelaskan bagaimana menjalankan `nuxt`:
```
Comment (jefrydco) [inline id/guide/installation.md]: I think the proper translation for 'specify' is 'menentukan' not 'menjelaskan'

File: id/guide/installation.md [hunk 3]
```diff
@@ -79,46 +104,46 @@ Proyek memerlukan file `package.json` untuk memperinci bagaimana untuk memulai `
...
 
 <div class="Alert">
 
-Nuxt.js akan memantau perubahan file di dalam direktori <code>pages</code>, jadi tidak perlu me-restart aplikasi saat menambahkan halaman baru.
+Nuxt.js akan memantau perubahan pada file anda yang berada dalam direktori <code>pages</code>, jadi anda tidak perlu menjalankan ulang aplikasi anda ketika menambahkan halaman baru.
 
 </div>
 
-Untuk mengetahui lebih lanjut tentang struktur direktori pada proyek: [Dokumentasi Struktur Direktori](/guide/directory-structure).
+Untuk menjelajahi lebih lanjut mengenai struktur direktori pada projek : [Directory Structure Documentation](/guide/directory-structure).
```
Comment (jefrydco) [inline id/guide/installation.md]: I think the proper translation for `to discover more about` is 'untuk menemukan lebih banyak' not 'menjelajahi lebih lanjut'

File: id/guide/installation.md [hunk 4]
```diff
@@ -16,59 +16,84 @@ description: Memulai Nuxt.js itu ternyata sangat mudah. Satu proyek sederhana ha
...
+- [Element UI](https://github.com/ElemeFE/element)
+- [Ant Design Vue](https://github.com/vueComponent/ant-design-vue)
+- [Buefy](https://buefy.github.io)
+
+3. [Mode Nuxt yang anda inginkan (`Universal` or `SPA`)](https://nuxtjs.org/guide/release-notes#better-spa-experience)
+4. Menambah [axios module](https://github.com/nuxt-community/axios-module) untuk mempermudah melakukan HTTP request pada aplikasi anda.
+5. Menambah [EsLint](https://eslint.org/) untuk memformat coding anda dengan lint pada saat menyimpan.
+6. Menambah [Prettier](https://prettier.io/) untuk memformat coding anda dengan prettier pada saat menyimpan.
+
+Setelah menjawab semua pertanyaan semua dependencies yang dibutuhkan akan terinstall dan langkah selanjut nya adalah menjalankan aplikasi dengan:
```
[the remark this answers is not in the document]
Comment (buddyy93) [reply id/guide/installation.md]: thank you for your review, i've updated the translations

File: id/guide/installation.md (hunk 1 above)
Comment (buddyy93) [reply id/guide/installation.md]: thank you for your review, i've updated the translations

File: id/guide/installation.md (hunk 3 above)
Comment (buddyy93) [reply id/guide/installation.md]: thank you for your review, i've updated the translations

**Comment by buddyy93** [comment]
what should i do now so the update can be applied?

**Comment by manniL** [comment]
Thank you for your contribution ☺️

**Pull request merged**